src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/ReadyToRunLibraryRootProvider.cs

```diff
@@ -37,6 +37,12 @@ public void AddCompilationRoots(IRootingServiceProvider rootProvider)
                         {
                             continue;
                         }
+
+                        if (method.IsMethodDefinition)
```

Comment:
Add a comment explaining why you added this.

Reply:
The comment actually already exists above. This is about not working with uninstantiated methods.